packages/jest-runner/errorUtils.test.ts

```diff
@@ -31,31 +35,72 @@ describe("ErrorUtils", () => {
     at fulfilled (/home/norbert/Code-Intelligence/jazzer.js/packages/jest-runner/dist/fuzz.js:58:24)
 `;
 
-	it("clean up jest runner frames", () => {
-		expect(cleanupJestRunner(stack)).toEqual(`Error:
+	beforeEach(() => {
+		error.stack = stack;
+	});
+
+	describe("clean up Jest runner frames", () => {
+		it("in errors", () => {
+			expect(cleanupJestError(undefined)).toBeUndefined();
+			const cleanedUp = cleanupJestError(error);
+			expect((cleanedUp || {}).stack).toEqual(`Error:
+    at /home/norbert/Code-Intelligence/jazzer.js/examples/jest_integration/integration.fuzz.js:27:3
+`);
+		});
+
+		it("in stacks", () => {
+			expect(cleanupJestRunnerStack(undefined)).toBeUndefined();
+			expect(cleanupJestRunnerStack(stack)).toEqual(`Error:
     at /home/norbert/Code-Intelligence/jazzer.js/examples/jest_integration/integration.fuzz.js:27:3
 `);
+		});
 	});
 
 	describe("remove stack frames", () => {
-		it("on top", () => {
-			expect(removeTopFrames(undefined, 1)).toBeUndefined();
-			expect(removeTopFrames(stack, 3)).toEqual(`Error:
+		describe("in errors", () => {
+			it("on top", () => {
```

Comment:
Maybe `on top of remaining`?  Wouldn't expect to pop stack frames based on the desc. here otherwise.

Reply:
changed